src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpRawlsDAO.scala

```diff
@@ -61,7 +61,7 @@ class HttpRawlsDAO( implicit val system: ActorSystem, implicit val executionCont
         case Right(srw) =>
           logger.info("admin workspace list got: " + srw.length + " raw workspaces")
           val published = srw.collect {
-            case rw:RawlsWorkspace if rw.attributes.getOrElse(LibraryService.publishedFlag, "false").toBoolean => rw
+            case rw:RawlsWorkspace if rw.attributes.getOrElse(LibraryService.publishedFlag, AttributeBoolean(false)).asInstanceOf[AttributeBoolean].value => rw
```

Comment:
This is lame-duck code anyway and will be removed once broadinstitute/rawls#552 lands, so don't put too much effort into it - therefore, I suggest taking the easiest route. But my desired functionality would be that if the Attribute is an AttributeBoolean(true), then true; if the Attribute is anything else (e.g. AttributeBoolean(false), AttributeString, AttributeNumber, etc) then false.